lua/telescope/builtin/__git.lua

```diff
@@ -397,9 +383,21 @@ git.status = function(opts)
   pickers
     .new(opts, {
       prompt_title = "Git Status",
+      entry_maker = opts.entry_maker,
       finder = initial_finder,
       previewer = previewers.git_file_diff.new(opts),
       sorter = conf.file_sorter(opts),
+      on_complete = {
+        function(self)
+          local lines = self.manager:num_results()
+          if lines == 0 then
+            utils.notify("builtin.git_status", {
+              msg = "No changes found",
+              level = "ERROR",
+            })
+          end
+        end,
+      },
```

Comment:
this part here is interesting, i've never thought about this, how we should handle things if we dont have any results with a oneshot finder.

do you think it makes sense to then also close the picker, if we havent found anything?

Reply:
@Conni2461 thanks for the feedback I have addressed your comments and updated the PR.